libsel4/sel4_plat_include/rk3288/sel4/plat/api/constants.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+#pragma once
+
+#ifdef HAVE_AUTOCONF
```

Comment:
HAVE_AUTOCONF is always set nowadays. My PR just got merged that removed this ifdef check everywhere, please drop it here also.